docs/api/06_events/create-event.mdx

```diff
@@ -161,8 +161,8 @@ api/v1/events
 | Attributes | Type | Description |
 | -----------| -----| ----------- |
 | **transaction_id** | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Unique ID identifying the event. <br></br>As it will be used for idempotency, it should be a unique identifier |
-| **subscription_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique subscription ID in Lago's application.<br></br>`subscription_id` is required if the customer has multiple subscriptions or if `customer_id` is not provided (in case there's only one subscription) |
-| **customer_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Customer unique identifier in your application.<br></br>If `subscription_id` is not given, `customer_id` is required if there is only one subscription attached to customer. For multiple subscriptions per customer this attribute is not enough |
+| **external_subscription_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique subscription ID in Lago's application.<br></br>`external_subscription_id` is required if the customer has multiple subscriptions or if `external_customer_id` is not provided (in case there's only one subscription) |
```

Comment:
Replace "Unique subscription ID in Lago's application." with "Unique subscription ID in your application."

Reply:
Thanks Mathieu - Updating the base branch about external id 👍